feat(base): generate new image with version 14.19.0

bregenspan opened the pull request:
This adds an image for the latest 14.x maintenance LTS release as of now.

The commit here was automatically generated by running: `yarn add:base -- 14.19.0` in keeping with the [contribution guidelines](https://github.com/cypress-io/cypress-docker-images/blob/afd6bb6a69cffa8c8015d478ea0f8dbe4fb25dcf/CONTRIBUTING.md). Is the shuffling around of content in the Markdown changelog expected?

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/cypress-io/cypress-docker-images?pullRequest=596) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/cypress-io/cypress-docker-images?pullRequest=596) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/cypress-io/cypress-docker-images?pullRequest=596) it.</sub>

bregenspan commented:
The new image builds correctly locally, but it seems like something is off with the auto-updated circle.yml (I either did something wrong or there's something off in the automation changes in https://github.com/cypress-io/cypress-docker-images/pull/590)

bregenspan commented:
Looks like there were some fixes for the config generation script in https://github.com/cypress-io/cypress-docker-images/pull/595/files . I just recreated the commit with the new version.

bregenspan commented:
> @bregenspan sorry to ask, but could you recreate this based off the latest `master`? I fixed the issue where the `README` formatting was off. Thanks!

@admah done! I'm not clear why but the generation process keeps removing a readme line for "cypress/base:16.13.2", added that back manually and also filled in the Node/Yarn version details.

bregenspan commented on base/CHANGELOG.md:
```diff
@@ -67,3 +67,4 @@
 | cypress/base:ubuntu18-node12.14.1 | 12.14.1 | Ubuntu 18.04.3   | [ubuntu18-node12.14.1](ubuntu18-node12.14.1) | 6.13.6      | 1.21.1       |
 | cypress/base:ubuntu19-node12.14.1 | 12.14.1 | Ubuntu 19.0.4    | [ubuntu19-node12.14.1](ubuntu19-node12.14.1) | 6.13.6      | 1.21.1       |
 | cypress/base:manjaro-node14.12.0  | 14.12.0 | Manjaro          | [manjaro-14.12.0](manjaro-14.12.0)           | 6.14.8      | 1.22.10      | 
+| cypress/base:14.19.0              | 14.19.0 | Debian           | [/14.19.0](14.19.0)                          | 6.14.16     | 1.22.17      |
```
I filled in last 2 columns with Yarn/NPM versions I'm seeing at Docker build time

bregenspan commented on base/README.md:
```diff
@@ -77,9 +79,9 @@ Image `cypress/base:12` is tagged [`latest`](https://hub.docker.com/r/cypress/ba
-
-> **Note** All Base Images install the latest versions of NPM & Yarn
+| cypress/base:manjaro-node14.12.0  | 14.12.0 | Manjaro          | [manjaro-14.12.0](manjaro-14.12.0)           | 6.14.8      | 1.22.10      | 
+| cypress/base:14.19.0              | 14.19.0 | Debian           | [/14.19.0](14.19.0)                          | 6.14.16     | 1.22.17      |
```
I filled in last 2 columns with Yarn/NPM versions I'm seeing at Docker build time

admah replied:
Thanks!

admah reviewed (approved):
Looks great! 🚀

admah commented:
> @admah done! I'm not clear why but the generation process keeps removing a readme line for "cypress/base:16.13.2", added that back manually and also filled in the Node/Yarn version details.

Thanks for reporting it. I just created #604 to track that issue.
